pkg/cli/mirror/list/operators.go

```diff
@@ -132,21 +131,21 @@ func (o *OperatorsOptions) Run(cmd *cobra.Command) error {
 		}
 		res, err := lc.Run(ctx)
 		if err != nil {
-			logrus.Fatal(err)
+			return err
 		}
 		if err := res.WriteColumns(o.IOStreams.Out); err != nil {
-			logrus.Fatal(err)
+			return err
 		}
 	case len(o.Catalog) > 0:
 		lp := action.ListPackages{
 			IndexReference: o.Catalog,
 		}
 		res, err := lp.Run(ctx)
 		if err != nil {
-			logrus.Fatal(err)
+			return errors.New("Failed to list operators, please check catalog name: " + o.Catalog)
```

Comment:
One nit and then lgtm -> Should probably return the upstream error as well with `fmt.Errorf` in case the problem is network or auth related.